Resources/Prototypes/Entities/Objects/Consumable/Food/burger.yml

```diff
@@ -43,7 +43,7 @@
     - map: ["foodSequenceLayers"]
   - type: FoodSequenceStartPoint
     key: burger
-    maxLayers: 20
+    maxLayers: 4
```

Comment:
literally 1984 why make it so low now

Reply:
![image](https://github.com/user-attachments/assets/ceb1d3a6-2e6e-44d1-898c-3cd5c7df5ede)

Reply:
still

Reply:
![image](https://github.com/user-attachments/assets/595cd9fd-b84f-40bf-84a6-1fd890559456)

Reply:
so make them increase in size, 20 ingredients = 20 tall burger (impossible to store except in boh)

Reply:
We don't have a normal functionality for editing item size yet, and adding it is not within the scope of this PR.

Reply:
we used to